x-pack/plugins/data_visualizer/public/application/common/components/field_data_row/action_menu/actions.ts

```diff
@@ -87,6 +87,7 @@ export function getActions(
             indexPatternId: indexPattern.id,
             fieldName: item.fieldName,
             contextualFields: [],
+            originatingApp: 'data_visualizer:exploreInMapsTitle',
```

Comment:
How about changing to just "data_visualizer"? Is there a constant for data_visualizer?